dnf5/commands/download/download.cpp

```diff
@@ -152,7 +155,18 @@ void DownloadCommand::run() {
     }
 
     if (!download_pkgs.empty()) {
-        download_packages(download_pkgs, destdir_option->get_value().c_str());
+        libdnf::repo::PackageDownloader downloader(ctx.base);
+
+        // for download command, we don't want to mark the packages for removal
+        downloader.force_caching(true);
+
+        for (auto & pkg : download_pkgs) {
+            downloader.add(pkg, destdir_option->get_value().c_str());
+        }
```

Comment:
I think it would be better to use configuration value directly - see https://github.com/rpm-software-management/dnf5/pull/521